apps/server/src/auth/strategy/google-strategy.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { PassportStrategy } from '@nestjs/passport';
+
+import { Strategy, VerifyCallback } from 'passport-google-oauth2';
+
+import getGoogleStrategyConfig from 'src/config/googlestrategy.config';
```

Comment:
Let's utilize path aliases for such imports using '@/config/googlestrategy.config'`

Reply:
Will be updated in next PR